pkg/network/event_windows.go

```diff
@@ -158,17 +159,13 @@ func FlowToConnStat(cs *ConnectionStats, flow *driver.PerFlowData, enableMonoton
 
 			case crsp == driver.ClassificationResponseHTTP:
 				if flow.HttpUpgradeToH2Accepted == 1 {
-					cs.Protocol = ProtocolHTTP2
+					cs.ProtocolStack = protocols.Stack{Application: protocols.HTTP2}
 				} else {
 					// could have missed the request.  Most likely this is just
 					// resetting the existing value
-					cs.Protocol = ProtocolHTTP
+					cs.ProtocolStack = protocols.Stack{Application: protocols.HTTP}
 				}
 			}
-		} else {
-			// one of
-			// ClassificationUnableInsufficientData, ClassificationUnknown, ClassificationUnclassified
-			cs.Protocol = ProtocolUnknown
```

Comment:
Why remove this entirely, instead of setting `cs.ProtocolStack = protocols.Stack{Application: protocols.Unknown}` ?

Reply:
No particular reason to be honest (the 2 should be equivalent), but now that you're asking I think I should probably preserve the original form, just for the sake of consistency. I'll update it!